test/powershell/Modules/PSDesiredStateConfiguration/MOF-Compilation.Tests.ps1

```diff
@@ -24,7 +24,9 @@ Describe "DSC MOF Compilation" -tags "CI" {
         }
 
         DSCTestConfig
-"@).Invoke()
+"@).Invoke() 
+
+        Test-Path -Path .\DSCTestConfig\localhost.mof | Should Be $True
```

Comment:
to verify a file exists, please use ``<path> | should exist` , it results it better errors

Reply:
Updated